src/containers/ProjectContentsContainer.jsx

```diff
@@ -50,16 +52,26 @@ class ProjectContentsContainer extends Component {
     this.setState({modalOpen: false});
   }
 
+  handleChange(event) {
+    this.setState({
+      translationText: event.target.value,
+    });
+  }
+
   handleClick(event) {
     this.setState({
+      field: event.target.textContent.split(':')[0],
```

Comment:
`event.target.getAttribute('data-translation-key')` would be better here. Then add that attribute to editable fields in the view eg. `data-translation-key='description'`. You started on that in #4.